bot/gwapi/facade.py

```diff
@@ -70,16 +75,19 @@ def guild_search(guild_name: str) -> Optional[str]:
     return search_result[0]
 
 
+@cached(cache=TTLCache(maxsize=32, ttl=300))  # cache clients for 5 min - creation takes quite long
 def account_get(api_key: str) -> Account:
     api = _create_client(api_key=api_key)
     return _check_error(api.account.get())
 
 
+@cached(cache=TTLCache(maxsize=32, ttl=300))  # cache clients for 5 min - creation takes quite long
 def characters_get(api_key: str) -> List[Character]:
     api = _create_client(api_key=api_key)
     return _check_error(api.characters.get(page="0", page_size=200))
```

Comment:
What is the reasoning behind these paging boundaries?

Reply:
If paging parameters are not supplied, the response does not include the details
https://api.guildwars2.com/v2/characters?page=0&page_size=200&access_token=564F181A-F0FC-114A-A55D-3C1DCD45F3767AF3848F-AB29-4EBF-9594-F91E6A75E015&lang=en

https://api.guildwars2.com/v2/characters?access_token=564F181A-F0FC-114A-A55D-3C1DCD45F3767AF3848F-AB29-4EBF-9594-F91E6A75E015&lang=en

Taken from https://status.gw2efficiency.com/

Reply:
Okay, so the number is basically arbitrary, got it.